runtime/compiler/runtime/J9ValueProfiler.hpp

```diff
@@ -68,6 +68,14 @@ typedef uint32_t ProfileAddressType;
 typedef uintptr_t ProfileAddressType;
 #endif
 
+/** 
+ * Clang does not consider uintptr_t to be equivalent to uint64_t
+ */
+
+#if defined(OSX)
```

Comment:
Can this be merged into the if / else logic above?  This will typedef ProfileAddressType twice.